test/databaseManager.js

```diff
@@ -44,55 +45,61 @@ describe('Data Base manager test', () => {
             administrator: user._id
         });
         await discussion2.save().then((discussionSaved) => {
-            console.log("discussion saved : ", discussionSaved);
+            logger.debug(`discussion saved : ${discussionSaved}`);
         });
 
         const question1 = new Question({
-            _id: 1,
+            id: 1,
             titreQuestion: "Question for debate 1",
             numberVotes: 0,
             refDiscussion: discussion1._id
         });
         await question1.save().then((questionSaved) => {
-            console.log("question saved : ", questionSaved);
+            logger.debug(`question saved : ${questionSaved}`);
         });
 
         const question2 = new Question({
-            _id: 2,
+            id: 2,
             titreQuestion: "Question 2 for debate 1",
             numberVotes: 0,
             refDiscussion: discussion1._id
         });
         await question2.save().then((questionSaved) => {
-            console.log("question saved : ", questionSaved);
+            logger.debug(`question saved : ${questionSaved}`);
         });
 
         const device = new Device({
             _id: "110e8400-e29b-11d4-a716-446655440000"
         });
         await device.save().then((deviceSaved) => {
-            console.log("device saved : ", deviceSaved);
+            logger.debug(`device saved : ${deviceSaved}`);
         });
 
         const response1 = new Response({
-            _id: 1,
+            id: 1,
             response: "Yes",
-            refQuestion: question1._id,
+            refQuestion: {
+                refQuestion: question1.id,
+                refDiscussion: discussion1._id
```

Comment:
Rename _id to id from Discussion class as well. It's possible you already did it in another PR. I will check for it.

Reply:
I saw that you did it in the PR #39. I'm not sure if you have updated all the call to discussion.id accordingly afterward ?

Reply:
I changed Question to id field because the default _id filed that is used as primary key in a MongoDB database cannot be used as a primary key composite.